packages/shared/components/organisms/AccountActivity.svelte

```diff
@@ -21,6 +25,58 @@
         })
     }
 
+    const handleSyncAccountClick = () => {
+        if (!$isSyncing) {
+            const _syncAccount = () => {
+                $isSyncing = true
+                api.syncAccount($selectedAccount?.id, {
+                    onSuccess() {
+                        $isSyncing = false
+                    },
+                    onError(err) {
+                        $isSyncing = false
+
+                        const shouldHideErrorNotification =
+                            err && err.type === 'ClientError' && err.error === 'error.node.chrysalisNodeInactive'
+                        if (!shouldHideErrorNotification) {
+                            if ($isLedgerProfile) {
+                                displayNotificationForLedgerProfile('error', true, true, false, false, err)
+                            } else {
+                                showAppNotification({
+                                    type: 'error',
+                                    message: localize(err.error),
+                                })
+                            }
+                        }
+                    },
+                })
+            }
+
+            if ($isSoftwareProfile) {
+                api.getStrongholdStatus({
+                    onSuccess(strongholdStatusResponse) {
+                        if (strongholdStatusResponse.payload.snapshot.status === 'Locked') {
+                            openPopup({
+                                type: 'password',
+                                props: { onSuccess: () => _syncAccount() },
+                            })
+                        } else {
+                            void _syncAccount()
+                        }
+                    },
+                    onError(err) {
+                        showAppNotification({
+                            type: 'error',
+                            message: localize(err.error),
+                        })
+                    },
+                })
+            } else {
+                void _syncAccount()
+            }
+        }
+    }
+
```

Comment:
This is using the old apis for getting the stronghold status and the syncing of accounts. We need to call sync on the selected account. And check the stronghold status for the profile-manager.